### DISCUSSION TITLE
Passing variables in scaledobject .

### DISCUSSION THREAD

**Discussion by Savitha-Amaldev** [kedacore/keda#2173] [Q&A / Need Help] [answered]
I'm using an MSSQL trigger in my scaled object and I want to pass the below values via a variable. How can I achieve that? 

 name: {scaled-object-name}
  pollingInterval: 30                          
  cooldownPeriod:  300                          
  minReplicaCount: 0                              
  maxReplicaCount: 100   
 query: "SELECT COUNT(*) FROM backlog WHERE state='running' OR state='queued'"      

My question is if I'm defining a property file, how does my scaled object correctly recognizes the actual value? I saw a couple of scripts where we do export and all. But didn't get a clear-cut picture of how to do that. Can someone please help me here?

**Comment by tomkerkhove** [comment, marked as answer]
That's something you can do with Helm charts to generate the Kubernetes resources based on your configuration.

**Reply by Savitha-Amaldev** [reply]
Can we use  Kustomize as well for passing the namespace variable?
